### PR TITLE
Fb asim 2265 dynamic mfd

### PR DESCRIPTION
**Pull request by tarcisiofischer**

### CODE REVIEW

File: alfasim_sdk/hook_specs_gui.py [hunk 1]
```diff
@@ -76,6 +74,45 @@ def alfasim_get_status(ctx):
...
+
+@hookspec
+def alfasim_configure_layers():
+    """
+    Configure new or update layers for ALFAsim's hydrodynamic model.
+    Example:
+
+    return [
+        AddLayer(name='plugin_layer', fields=['plugin_continuous_field', DROPLET_FIELD], continuous_field='plugin_continuous_field'),
+        UpdateLayer(name=WATER_FIELD, additional_fields=['plugin_dispersed_field']),
```
Comment (ggrbill) [inline alfasim_sdk/hook_specs_gui.py]: Don't you mean `WATER_LAYER` instead of `WATER_FIELD`?

File: alfasim_sdk/hook_specs_gui.py [hunk 2]
```diff
@@ -76,6 +74,45 @@ def alfasim_get_status(ctx):
...
+
+
+@hookspec
+def alfasim_configure_phases():
+    """
+    Configure new or update phases for ALFAsim's hydrodynamic model.
+    Example:
+
+    return [
+        AddPhase(name='plugin_phase', fields=['plugin_continuous_field', DROPLET_FIELD], primary_field='plugin_continuous_field'),
```
Comment (ggrbill) [inline alfasim_sdk/hook_specs_gui.py]: I don't think add a phase with pre-existing fields is a good idea. If possible, please remove `DROPLET_FIELD` from example.

File: alfasim_sdk/hook_specs_gui.py (hunk 1 above)
Comment (williamjamir) [reply alfasim_sdk/hook_specs_gui.py]: I need to merge this PR to fix the master on ALFAsim, I created an issue for us to deal with it later, it's okay for you?

File: alfasim_sdk/hook_specs_gui.py (hunk 2 above)
Comment (williamjamir) [reply alfasim_sdk/hook_specs_gui.py]: I need to merge this PR to fix the master on ALFAsim, I created an issue for us to deal with it later, it's okay for you?

**Pull request merged**